utils/2.15.X-fix_hypertable_foreign_keys.sql

```diff
@@ -0,0 +1,77 @@
+CREATE OR REPLACE FUNCTION pg_temp.constraint_columns(regclass, int2[]) RETURNS text[] AS
```

Comment:
-- Fix compressed hypertables with FOREIGN KEY constraints that were created with TimescaleDB versions before 2.15.0